src/cljs/org/broadinstitute/firecloud_ui/page/workspace/analysis/track_selector.cljs

```diff
@@ -0,0 +1,156 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.analysis.track-selector
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.entity-table :refer [EntityTable]]
+    [org.broadinstitute.firecloud-ui.common.modal :as modal]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.page.workspace.analysis.igv-utils :as igv-utils]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
+
+
+(def ^:private supported-file-types [".bam" ".vcf" ".bed"])
```

Comment:
Hmm ... after talking to a few people, we will likely need to iterate on this with PO. VCFs can be gzipped, bzipped, etc ... leading to different file extensions (and paths to indexes). Certainly not for this PR.